Sources/Rendering/Core/ImageMapper/index.d.ts

```diff
@@ -34,10 +34,11 @@ interface IImageMapperInitialValues {
 
 export interface vtkImageMapper extends vtkAbstractMapper {
 
-	/**
-	 *
-	 * @param {Number} pos
-	 */
+  /**
+   * Get the slice index from pos.
+   * When slicing with XYZ, returns intermediary slice positions instead of IJK indices.
+   * @param {number[] | number} [pos] World point or XYZ slice value
+   */
 	getSliceAtPosition(pos: number): number;
 
```

Comment:
`getSliceAtPosition` is meant to convert a world position to a slice index. The current slicing mode is only used when passing a single number, so it goes from 1d to 3d based on the current axis. This method should not be used to pass coordinates in index space.

**A couple of suggestions:**
1. Improve current doc to illustrate `getSliceAtPosition` 
2. Consider renaming? (`getSliceIndexAtWorldPosition`)
3. Consider a separate method to get the continuous index so we don't break too many assumptions in apps that depend on `getSliceAtPosition` ? Similar to ITK (ex: `getContinuousSliceIndexAtWorldPosition` ... ugh). No strong feelings on this atm.